resources/migrations/000_migrations.yaml

```diff
@@ -3670,3 +3670,68 @@ databaseChangeLog:
                   name: result_metadata
                   type: text
                   remarks: 'Serialized JSON containing metadata about the result columns from running the query.'
+  - changeSet:
+      id: 58
+      author: senior
+      changes:
+        - createTable:
+            tableName: dimensions
```

Comment:
All of our other table names are singulars of the entity that they store attributes of. e.g. `pulse` instead of `pulses`